config/routes.rb

```diff
@@ -1,5 +1,8 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
   namespace :api do
     resources :articles, only: %i[index show], constraints: { format: 'json' }
+    post 'articles/create'
```

Comment:
please remove this and add `:create` to the resources helper above

Reply:
done!